### ISSUE TITLE
Do you know MinimaLT?

### ISSUE THREAD

**Issue by grd** [treeform/netty#27]
https://cr.yp.to/tcpip/minimalt-20130522.pdf

It is a paper only about a secure network protocol based on UDP. There are also a couple of implementations:

https://github.com/dcegielka/minimalt  (in C)
https://github.com/yorickvP/MinimaLT ( in JS)

And I thought one in Go as well but I can't find it right now.

**Comment by treeform** [comment]
No, sorry, I don't. MinimaLT is mostly about security and cryptography, while netty is about simplicity and low-latency reliability for games.
